### DISCUSSION TITLE
Nosgx

### DISCUSSION THREAD

**Discussion by Jmotyka23** [elementary/triage#173] [Q&A]
I have an Asus GU502 I have tried installing with multiple BIOS setting changes including "Secure Boot" enable and disabled and still get a SGX disable in bios error. I have tried adding "nosgx" to grub but it says not found. Any help would greatly be appreciated.

**Comment by DanielDO-WML** [comment]
try adding nosgx like this:
GRUB_CMDLINE_LINUX_DEFAULT="quiet splash"
GRUB_CMDLINE_LINUX="nosgx" <--

**Comment by NFEL** [comment]
If It helps , I have asus ux501vw and it used to not boot with elemntary os installed versions 5 , 5.1 and it failed to boot.
Issue was fixed in 6.1 release by installing os with live boot @Jmotyka23 Jmotyka23
